Assignment2/AlvaroMK-Alvaro Ajenjo Jurado-fromRDFtoJSON-LD.jsonld.txt

```diff
@@ -0,0 +1,59 @@
+{
+	"@context":{
+		"base": "http://example.org/",
+		"xsd": "http://www.w3.org/2001/XMLSchema#",
+		"Class01": "http://example.org/components/Class01",
+		"Sensor029": "http://example.org/components/Class01/Sensor029",
+		"Computer101": "http://example.org/components/Class01/Computer101",
+		"Measurement8401": "http://example.org/components/Class01/Sensor029/Measurement8401",
+		"User10A": "http://example.org/components/Class01/Computer101/User10A",
+		"includes": "http://example.org/components/properties/includes",
+		"hasMeasurement": "http://example.org/components/properties/hasMeasurement",
+		"hasOwner": "http://example.org/components/properties/hasOwner",
+		"hasTemperature": "http://example.org/components/properties/hasTemperature",
+		"atTime": "http://example.org/components/properties/atTime",
+		"hasOwner": "http://example.org/components/properties/hasOwner",
+		"hasName": "http://example.org/components/properties/hasName"
+	},
+	"@id": "http://example.org/components/Class01",
+	"@type": "Class",
+	"includes": {
+
+		"Sensor029": {

```

Comment:
You don't need to define "Sensor029": because it isn't an attribute, it is a resource, thus I suggest you remove this name. Remember that when you define a resource, you only have to use curly braces and define inside all the resource attributes. However, when you have to specify a relationship between resources is when you have to define it as an attribute, for example as you did with the attribute "hasMeasurement" for defining the relationship between Sensor029 and Measurment8401. 
This comment also applies for "Measurement8401":, "Computer101":, and "User10A":.